lib/services/audit_webhook.rb

```diff
@@ -9,8 +9,7 @@ class AhaServices::AuditWebhook < AhaService
 
   boolean :validate_cert,
     description: "Validate your server's HTTPS/TLS certificate",
-    label_name: "Validate certificate",
-    default: "1"
```

Comment:
Using `default` like this doesn't actually do anything for v1 services. The correct way to do this is in aha-app